docker-images/base/Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG TARGETARCH
 USER root
 
 RUN microdnf update \
-    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-${JAVA_VERSION}-openjdk-headless openssl shadow-utils \
+    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install java-${JAVA_VERSION}-openjdk-headless openssl shadow-utils bind-utils \
```

Comment:
I think you should install this only in the Kafka container image used for ZooKeeepr. Not to all of them.

Reply:
Yeah good point, agree.